impact.tex

```diff
@@ -460,55 +460,68 @@ \subsubsection{Dissemination of results}
 \end{enumerate}
 
 Our focus for dissemination will be on
-% \taskref{education}{workshops},
+\taskref{education}{workshops}
 operating workshops, training various communities in the availability,
 purpose, development, and use of \TheProject software and services.
 We will make a particular effort to use these workshops as an opportunity
 to \textbf{support diversity and inclusion in the Open Science community},
 by running (online, hybrid or in-person) workshops for under-served and under-represented groups in the academic and
-open source communities. \textbf{Free and online training} will be streamed (Twitch),  
+open source communities. \textbf{Free and online training} will be streamed (Twitch),
  make available to the wider community for instance on YouTube and archived on Zenodo for long-term availability and
- findability to the wider community who may not be able to attend workshops. 
+ findability to the wider community who may not be able to attend workshops.
+
 
 These resources will be hosted on free, public hosting services,
-such as GitHub Pages, YouTube channels and as much as possible co-developed and co-hosted with existing and 
+such as GitHub Pages, YouTube channels and as much as possible co-developed and co-hosted with existing and
 well-established organizations
 (The Carpentries, CodeRefinery, Galaxy Training Network, Pangeo, etc.).
 
 
 The operation of prototype deployments of binder services
```

Comment:
This is a duplicated paragraph (see next comment). Any preference which of the 2 nearly identical versions you'd like to keep?